src/libdrakvuf/vmi.c

```diff
@@ -674,6 +676,7 @@ event_response_t cr3_cb(vmi_instance_t vmi, vmi_event_t* event)
     drakvuf_trap_info_t trap_info;
     memset(&trap_info, 0, sizeof(drakvuf_trap_info_t));
 
+    trap_info.os = vmi_get_ostype(vmi);
```

Comment:
No need to store the os type in each `trap_info`, we already have `drakvuf_get_os_type`

Reply:
@tklengyel, thank you for note. Fixed.